antarest/launcher/repository.py

```diff
@@ -40,3 +40,13 @@ def delete(self, id: str) -> None:
         g = db.session.query(JobResult).get(id)
         db.session.delete(g)
         db.session.commit()
+
+    def delete_by_study_id(self, study_id: str) -> None:
+        job_results_to_delete: List[JobResult] = (
```

Comment:
add some log

Reply:
There are no logs in the whole class, i think the logs should either be put somewhere else, or should be added in all the methods of the class

Reply:
yes, logs should be added in the service.
Yet, since this method is called directly from the on_delete callback, it should then be wrapped within a service method with a log.

Reply:
also, it would be better to add the callback from the launcher service init constructor